src/temporary_home/RequestHandler.h

```diff
@@ -29,6 +29,10 @@ class RequestHandler {
       StreamId streamId,
       const yarpl::Reference<yarpl::flowable::Subscriber<Payload>>& response) noexcept = 0;
 
+  /// Handles a Stream resumed after cold restart
+  virtual yarpl::Reference<yarpl::flowable::Subscriber<Payload>> handleResumeStream(
```

Comment:
I would prefer passing this callback in the resume() call instead of having it here, but we can fix that up later